README.md

```diff
@@ -117,6 +117,17 @@ In default-mode, the script is not building the provided test. For a clean build
 
 If the script is not used, keep in mind to pass `-DTOML_CONFIG=on` to cmake if the optional RouDi config file feature shall be built.
 
+**NOTE:** Users who wish to build iceoryx with cmake can follow the below steps (Requires CMake 3.14 or higher).
```

Comment:
is cmake 3.14 really required?

Reply:
For building the project using iceoryx_meta, cmake 3.14 is required (for building with script )

Reply:
this should be a temporary situation. I've a patch somewhere to make this build with cmake 3.5 ... need to remove the dust from it, though

Reply:
For now I would leave it as it is since this is not our default build process ... not yet, but It should be.